File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int s=0;
        int e=size(numbers)-1;
    
        while(s<e)
        {
            int sum= numbers[s]+numbers[e];
            if(sum==target){
                return{s+1,e+1};

            }
            else if(sum>target)
            {
                e=e-1;
            }
            else{
                s=s+1;
            }
        }
        return {};
    }      
    
};

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        n=len(numbers)
        s=0
        e=n-1
        while(s<=e):
            summ = numbers[s]+numbers[e]
            if(summ == target):
                return [s+1,e+1]
            elif(summ > target):
                e=e-1
            else:
                s=s+1
            
        



        